### PR TITLE
Ensure timestamps are returned in workspace serializer

### PR DESCRIPTION
**Pull request by coderbydesign**
## Link(s) to Jira
- https://issues.redhat.com/browse/RHCLOUD-35150

## Description of Intent of Change(s)
Currently we have our base workspace model defined in the API spec as having a "created" and "modified" field, however those fields are not returned on list/details endpoints. This fixes that by adding them to the serializer.

## Local Testing
Using the `GET /workspaces/` and `GET /workspaces/<uuid>/` API endpoint locally, confirm `created` and `modified` now exist on the payload, per the spec.

## Checklist
- [ ] if API spec changes are required, is the spec updated?
- [ ] are there any pre/post merge actions required? if so, document here.
- [ ] are theses changes covered by unit tests?
- [ ] if warranted, are documentation changes accounted for?
- [ ] does this require migration changes?
  - [ ] if yes, are they backwards compatible?
- [ ] is there known, direct impact to dependent teams/components?
  - [ ] if yes, how will this be handled?

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Practices Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

**Review by vbelchio — APPROVED** [review]
LGTM

File: rbac/management/workspace/serializer.py [hunk 1]
```diff
@@ -28,12 +28,21 @@ class WorkspaceSerializer(serializers.ModelSerializer):
...
     parent_id = serializers.UUIDField(allow_null=True, required=False)
+    created = serializers.DateTimeField(read_only=True)
+    modified = serializers.DateTimeField(read_only=True)
 
     class Meta:
         """Metadata for the serializer."""
 
         model = Workspace
-        fields = ("name", "uuid", "parent_id", "description")
+        fields = (
```
Comment (astrozzc) [inline rbac/management/workspace/serializer.py:38]: '__all__' will also work in this case 😁

**Review by astrozzc — APPROVED** [review]
LGTM！

**Review by lpichler — APPROVED** [review]

File: rbac/management/workspace/serializer.py (hunk 1 above)
Comment (astrozzc) [reply rbac/management/workspace/serializer.py:38]: Right, sounds good, just leave it as it is 👍

Comment (coderbydesign) [reply rbac/management/workspace/serializer.py:38]: Thanks for the feedback!

**Pull request merged**